python/tests/integration/arcticdb/version_store/test_snapshot.py

```diff
@@ -595,37 +594,32 @@ def test_delete_snapshot_basic_flow_with_delete_prev_version(basic_store):
     versions_initial = sorted([v["version"] for v in lib.list_versions()])
 
     assert sorted(lib.list_snapshots()) == [snap1, snap2] 
-    assert df_combined.equals(lib.read(symbol).data)
+    assert_frame_equal(df_combined, lib.read(symbol).data)     
     assert versions_initial == [0, 1]
 
     lib.delete_version(symbol, 0)
 
     assert sorted(lib.list_snapshots()) == [snap1, snap2] 
     # list_versions() return the newest versions first
-    assert lib.list_versions()[0]["version"] == 1
-    assert not lib.list_versions()[0]["deleted"]
-    assert lib.list_versions()[-1]["version"] == 0
-    assert lib.list_versions()[-1]["deleted"]
-    assert df_combined.equals(lib.read(symbol).data)
+    
+    assert_frame_equal(df_combined, lib.read(symbol).data)     
```

Comment:
You seem to have dropped the assertions on the `list_versions` here? I.e. something like
```
assert [ver["version"] for ver in lib.list_versions()] == [1, 0]
assert [ver["deleted"] for ver in lib.list_versions()] == [False, True]
```

Reply:
done